doc/user_guide/examples.rst

```diff
@@ -397,19 +489,50 @@ code that is output is the same as the original (but looks different
 as it has been translated to PSyIR and then output by the PSyIR
 fortran back-end).
 
-Example 16: Kernel Extraction
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The subdirectory ``full_example_extract`` contains a runnable example
-of LFRic code that creates a NetCDF file with the input and output
-parameters of a simple kernel. This example requires the installation
-of a Fortran compiler and NetCDF development environment. After compilation
-of the code, you can run the example. which will produce a NetCDF file:
+Example 17: Runnable Simplified LFRic Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This directory contains three simplified LFRic examples, that can be
+compiled and executed - a suitable Fortran compiler is of course
+required. The examples are using a subset of the LFRic infrastructure
+library, which is contained in PSyclone and which has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
+
+Example 17.1: Very Simple Runnable Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The subdirectory ``full_example`` contains a very simple example code
+that uses PSyclone to process two invokes. It uses unit-testing
+code from various classes to create the required data structures like
+initial grid etc. You can compile the code with ``make compile``, and
+execute the binary with either ``make run`` or ``./example``
+
+Example 17.2: Very Simple Runnable Example With NetCDF
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The subdirectory ``fulf_example_netcdf`` contains code very similar
+to the previous example, but uses NetCDF to read the initial grid
+from the NetCDF file ``mesh_BiP128x16-400x400.nc``.
```

Comment:
You are right about the mesh file name, it should be ``mesh_BiP128x16-400x100.nc``!

Reply:
All renamed.

Reply:
Thanks!